### DISCUSSION TITLE
Condensed Reblogs: additional options

### DISCUSSION THREAD

**Discussion by marcustyphoon** [AprilSylph/XKit-Rewritten#109] [Suggestion Box]
"Trim leading items when the post has an addition" feature:
- Trim all but [number] items of context before the addition (default: 1)
- Trim the original post (default: on...?)

I recognize that I'm not in the main audience for this script, but when I did try it out and tried to imagine using it extensively for RP, the two things I thought I'd want were ~2-3 visible context entries rather than just one, and perhaps the ability to show the original post before the button. Right now it seems like it could be a bit hard to tell what the context is without clicking expand in some cases, which defeats the point. And for non-RP use, I think I'd definitely use the second option.

Based on a quick look at the current code these seem feasible to do without fundamentally changing much, I think?

**Comment by AprilSylph** [comment]
My understanding is that most roleplay on Tumblr is very long-form, and roleplayers don't keep more than one context addition when trimming with Editable Reblogs in XKit 7. I know they're also not happy about no longer being able to keep post titles either way, but I don't think showing the entire original trail item is the solution to that.

**Comment by honeyrp** [comment]
To offer the perspective of someone who does primarily use tumblr for rp, I think my biggest problem with condensed reblogs is that it's not a two way change. It only changes the view of the post if you have xkit rewritten, and it's only on the desktop dashboard. 

A really key feature to editable reblogs if that when I edit a post to "cut" post content is that that change is uniform regardless of where or how you're viewing it. I read and check on my dash a lot from the mobile app, and to my understanding, it would mean that these condensed posts would appear fully in the app? And on most themes.

I completely understand that a comparable feature with editable reblogs with the new post editor may not be achievable and I really appreciate the work that's gone into XKit Rewritten, but I did want to share why the community isn't fond of condensed reblogs replacing that feature in it's current state because to me, they don't achieve the same function.

**Reply by AprilSylph** [reply]
I do understand that Condensed Reblogs isn't perfect, and I have been thinking about ways to actually remove parts of the reblog trail. That's as far as the API supports, though - removing whole items - so it's not going to be *Editable* Reblogs. I could probably build a relatively simple UI for it, but it'll still be a lot of work, so think of Condensed Reblogs as a stopgap until this theoretical "Reblog Trimmer" feature comes out.

**Reply by AprilSylph** [reply]
Trim Reblogs was implemented in 7a9a8a7701730055d82ec7cd4f4979b3ae01d346 and released in v0.8.0, with Condensed Reblogs being removed at the same time.

(I really need to keep on top of older discussions better.)
